ci-jobs/build.yml

```diff
@@ -9,4 +9,9 @@ jobs:
           addChangeLog: false
   - template: ./templates/build.yml
     parameters:
-      name: 'macOS_10_14'
\ No newline at end of file
+      excludeXcode: '11.1,11.2.1,11.2,11.3.1,11.3,11'
```

Comment:
why it is necessary to explicitly exclude these versions? Azure selects the most recent version by default, doesn't it?

Reply:
@mykola-mokhnach sorry for the super late reply.

We only want one build per iOS version so we exclude versions from 10.14 that are included in 10.15

Reply:
I'll add a comment.